src/main/java/kr/codesqaud/cafe/controller/UserController.java

```diff
@@ -47,25 +48,57 @@ public String userProfile(Model model, @PathVariable String id) {
     }
 
     @GetMapping("/users/{id}/form")
-    public String userProfileCorrection(Model model, @PathVariable String id) {
+    public String userProfileCorrection(Model model, @PathVariable String id, HttpSession session) {
         Optional<User> user = userRepository.findById(id);
+
         if (user.isPresent()) {
-            model.addAttribute("user", user.get());
-            return "user/updateForm";
+            Object value = session.getAttribute("loginUser");
```

Comment:
```suggestion
            User user = (User) session.getAttribute("loginUser");
```

"loginUser" 는 상수로 처리 
 (User) session.getAttribute("loginUser");  는 자주 사용되니 별도 메소드로 구현하면 좋지 않을까요?